gcn_classic_text_to_json/notices/calet/README.md

```diff
@@ -0,0 +1,21 @@
+# CALET Text Conversion
+
+### Uses the following fields from the core schema for text notice fields
+- `id` &#8594; TRIGGER_NUM
+- `ra` &#8594; POINT_RA
+- `dec` &#8594; POINT_DEC
+- `alert_datetime` &#8594; NOTICE_DATE
+- `trigger_time` &#8594; TRIGGER_DATE, TRIGGER_TIME
+- `rate_snr` &#8594; SIGNIFICANCE
+- `latitude`, `longitude` &#8594; SC_LON_LAT
+- `rate_energy_range` &#8594; ENERGY_BAND
+- `detector_status` &#8594; TRIGGER_DET
+
+### Defines the following new fields for the text notice fields
+- `url` &#8594; LC_URL,
+- `rate_snr` &#8594; SIGNIFICANCE
+- `foreground_duration` &#8594; FOREGND_DUR
+- `background_duration` &#8594; BACKGND_DUR1
+
+## TBD
+- Update `url` with an accurate URL
```

Comment:
What does "the accurate URL" mean here? Is it parsed from a field?

Reply:
The complete discussion is [here](https://github.com/nasa-gcn/gcn-classic-to-json/issues/34), but the TLDR is that the I found from the source code that the url provided in the CALET notices is a placeholder. I think the plan is to reach out to the mission to see if they an accurate url now, but in the mean time @Vidushi-GitHub wanted me to record any incomplete conversions in a README.

Reply:
I've updated the README to better represent this issue